contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViews.java

```diff
@@ -138,6 +138,22 @@
           RpcViewConstants.GRPC_SERVER_SENT_MESSAGES_PER_METHOD_VIEW,
           RpcViewConstants.GRPC_SERVER_RECEIVED_MESSAGES_PER_METHOD_VIEW);
 
+  @VisibleForTesting
+  static final ImmutableSet<View> GRPC_CLIENT_BASIC_VIEWS_SET =
+      ImmutableSet.of(
+          RpcViewConstants.GRPC_CLIENT_ROUNDTRIP_LATENCY_VIEW,
+          RpcViewConstants.GRPC_CLIENT_STARTED_RPC_VIEW,
+          RpcViewConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW,
```

Comment:
Also add `RPC_CLIENT_STARTED_COUNT_CUMULATIVE_VIEW `.

Reply:
it is there. see below.